### ISSUE TITLE
Why disable shuffle in train dataloader?

### ISSUE THREAD

**Issue by Luciennnnnnn** [cswry/SeeSR#29]
Why disable shuffle in train dataloader? It is strange since random shuffle in training is standard practice. Furthermore, I try to finetune SeeSR in my data, the results become even worse than pretrained SeeSR, I use same training as yours, except with my own data.

**Comment by zhouyizhuo** [comment]
Have you discovered the reason?

**Comment by cswry** [comment]
> Why disable shuffle in train dataloader? It is strange since random shuffle in training is standard practice. Furthermore, I try to finetune SeeSR in my data, the results become even worse than pretrained SeeSR, I use same training as yours, except with my own data.

Hello, since we prepared multi-epochs of training data offline, we believe that setting shuffle to True or False during online training should not make a difference. If you find that the model performs differently when the shuffle setting is different, please feel free to report your findings.
